<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QByJobs</class>
 <widget class="QMainWindow" name="QByJobs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>工单（&amp;J）:</string>
            </property>
            <property name="buddy">
             <cstring>comboBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>ZZ</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>开始时间（&amp;S）：</string>
            </property>
            <property name="buddy">
             <cstring>start</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="start">
            <property name="displayFormat">
             <string>yyyy年M月d日 HH:mm:ss</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>结束时间（&amp;E）:</string>
            </property>
            <property name="buddy">
             <cstring>end</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="end">
            <property name="displayFormat">
             <string>yyyy年M月d日 HH:mm:ss</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>工具（&amp;T）:</string>
            </property>
            <property name="buddy">
             <cstring>toolBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="toolBox">
            <item>
             <property name="text">
              <string>ZZ20190130001</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZZ20190131002</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZZ20190201001</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZZ20190201002</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZZ20190201003</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZZ20190201004</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZZ20190204001</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZZ20190216001</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>打印</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>工人（&amp;W）：</string>
            </property>
            <property name="buddy">
             <cstring>workBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="workBox">
            <item>
             <property name="text">
              <string>白一</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>洪二</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>張三</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>start</tabstop>
  <tabstop>end</tabstop>
  <tabstop>workBox</tabstop>
  <tabstop>toolBox</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
